src/main/java/tech/jhipster/lite/cli/command/infrastructure/primary/ModuleSlugCommand.java

```diff
@@ -0,0 +1,155 @@
+package tech.jhipster.lite.cli.command.infrastructure.primary;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.Callable;
+import picocli.CommandLine.ExitCode;
+import picocli.CommandLine.Model.CommandSpec;
+import picocli.CommandLine.Model.OptionSpec;
+import tech.jhipster.lite.cli.shared.error.domain.Assert;
+import tech.jhipster.lite.cli.shared.generation.domain.ExcludeFromGeneratedCodeCoverage;
+import tech.jhipster.lite.module.application.JHipsterModulesApplicationService;
+import tech.jhipster.lite.module.domain.JHipsterModuleSlug;
+import tech.jhipster.lite.module.domain.JHipsterModuleToApply;
+import tech.jhipster.lite.module.domain.properties.JHipsterModuleProperties;
+import tech.jhipster.lite.module.domain.properties.JHipsterPropertyDescription;
+import tech.jhipster.lite.module.domain.properties.JHipsterPropertyKey;
+import tech.jhipster.lite.module.domain.properties.JHipsterPropertyType;
+import tech.jhipster.lite.module.domain.resource.JHipsterModuleOperation;
+import tech.jhipster.lite.module.domain.resource.JHipsterModulePropertiesDefinition;
+import tech.jhipster.lite.module.domain.resource.JHipsterModuleResource;
+
+class ModuleSlugCommand implements Callable<Integer> {
```

Comment:
How about this naming?
```suggestion
class ApplyModuleSubCommand implements Callable<Integer> {
```

Reply:
Cool! Explicitly show the class proposal :)

Reply:
Cool! Explicitly show the class proposal :)

Reply:
Done :)